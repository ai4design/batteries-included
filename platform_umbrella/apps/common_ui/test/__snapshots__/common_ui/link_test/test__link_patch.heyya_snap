<a href="/test" data-phx-link="patch" data-phx-link-state="push" class="">
  Test patch
</a>