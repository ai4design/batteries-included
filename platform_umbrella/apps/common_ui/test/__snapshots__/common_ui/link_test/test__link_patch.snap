"<a href=\"/test\" data-phx-link=\"patch\" data-phx-link-state=\"push\" class=\"\">\n  Test patch\n</a>"